packages/linux-drivers/RTL8723BS_BT/package.mk

```diff
@@ -0,0 +1,55 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016-2017 Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="RTL8723BS_BT"
+PKG_VERSION="BOXSTCK1A8LFC"
```

Comment:
One thing... can you change the `PKG_VERSION` to a specific rev, such as `6a6f1da`.

Using a branch label as version means everyone will be building with different source code (depending on when they download the tarball, and won't receive any updates) should you check in any additional commits.